website/docs/docs/deploy/merge-jobs.md

```diff
@@ -14,26 +14,53 @@ By using CD in dbt Cloud, you can take advantage of deferral to build only the e
 - You have set up a [connection with your Git provider](/docs/cloud/git/git-configuration-in-dbt-cloud). This integration lets dbt Cloud run jobs on your behalf for job triggering.
    - If you're using a native [GitLab](/docs/cloud/git/connect-gitlab) integration, you need a paid or self-hosted account that includes support for GitLab webhooks and [project access tokens](https://docs.gitlab.com/ee/user/project/settings/project_access_tokens.html). If you're using GitLab Free, merge requests will trigger CI jobs but CI job status updates (success or failure of the job) will not be reported back to GitLab.
```

Comment:
Why do we only single out Gitlab here but not talk about setup for Github or ADO?